texthero/_types.py

```diff
@@ -43,8 +42,8 @@ def tfidf(s: TokenSeries) -> RepresentationSeries:
 - TextSeries: cells are text (i.e. strings), e.g. "Test"
 - TokenSeries: cells are lists of tokens (i.e. lists of strings), e.g. ["word1", "word2"]
 - VectorSeries: cells are vector representations of text, e.g. [0.25, 0.75]
-- RepresentationSeries: Series is multiindexed with level one
-being the document, level two being the individual features and their values
+- DataFrame: a Pandas DataFrame representing a matrix. Every cell value
+    is one entry in the matrix.
 
 The classes are lightweight subclasses of pd.Series and serve 2 purposes:
```

Comment:
Overall, the introduction is super great but not anymore 100% accurate.

I would add a separate paragraph about the new changes this PR make:

"Sometimes texthero functions also accept (standard) Pandas `DataFrame`s (in general, when they accept `VectorSeries`, is there any exception?). For this reason HeroSeries subclass also DataFrame ... "

Plus, would drop the "DataFrame: a Pandas ..." item from "These are the implemented types" as is not really an implementation.

Reply:
will change this